packages/mdc-textfield/foundation.js

```diff
@@ -65,6 +75,10 @@ export default class MDCTextfieldFoundation extends MDCFoundation {
     this.adapter_.registerInputBlurHandler(this.inputBlurHandler_);
     this.adapter_.registerInputInputHandler(this.inputInputHandler_);
     this.adapter_.registerInputKeydownHandler(this.inputKeydownHandler_);
+    this.adapter_.registerTransitionEndHandler(this.transitionEndHandler_);
+    ['mousedown', 'touchstart'].forEach((evtType) => {
```

Comment:
Just wanted to make sure we'd covered all the appropriate events. Looks like you have.